packages/debug/src/browser/debug-configuration-manager.ts

```diff
@@ -189,7 +193,23 @@ export class DebugConfigurationManager {
         }
     }
     async addConfiguration(): Promise<void> {
-        const { model } = this;
+        let model: DebugConfigurationModel | undefined;
+        if (this.models.size > 1) {
+            const rootUris = (await this.workspaceService.roots).map(r => r.uri);
```

Comment:
I don't this first `map` is needed: why not just use `r.uri` inside the quick pick?